### DISCUSSION TITLE
How to get the save data?

### DISCUSSION THREAD

**Discussion by Zandor94** [wolveix/satisfactory-server#165] [General]
Hello wolveix,

I have the problem that the config is not linked to a local folder and I can't find the save file.

There is one world running on the server and this world is also saved. It is also present every time you start it. Unfortunately I don't know how to find this world file on the Synology.

I urgently ask for help.

Many greetings

**Comment by wolveix** [comment]
Hi @Zandor94. Not to worry, it's easy to access the saves. I'd very strongly recommend mapping the directory as the README explicitly outlines in the future.

Despite the config directory not being mapped, the paths still work the same. Do something like this:

```
docker cp satisfactory:/config/saved .
```

This will copy your saved data to the directory you run the command from.

**Reply by Zandor94** [reply]
Many thanks for your response.

I was able to run the command. Thanks
